src/js/angular/core/services.js

```diff
@@ -286,3 +287,25 @@ function ClassInstanceDetailsService($http) {
         }
     }
 }
+
+function AuthTokenService() {
```

Comment:
It is a good idea to introduce an authentication service, but it would be better if you use the `LocalStorageAdapter` instead of `localStorage`. For example, you can see how the `WorkbenchSettingsStorageService` (workbench-settings-storage-service.js) is implemented.

Reply:
Well, there is an authentication service, it is called `$jwtAuth`, but it uses `$http` and having this inside causes a cyclic dependency. This is why I extracted it as a separate service.

As far as using the `LocalStorageAdapter`, I was going to but it prefixes all keys and as this key in particular is used separate, I didn't want to change it, so I left it like that.

The key is used by other libraries when they make requests. `swagger-ui.js` is one of them and it does not have access to `LocalStorageAdapter` so if the prefix is changed in the workbench config, it would stop working.